examples/SmartRefrigerator/SmartRefrigerator_ExplorerKit_Salesforce.agent.nut

```diff
@@ -26,7 +26,8 @@
 #require "Rocky.class.nut:1.2.3"
 
 // Web Integration Library
-#require "Salesforce.agent.lib.nut:2.0.0"
+// #require "Salesforce.agent.lib.nut:2.0.0"
```

Comment:
May want to switch back to require statement before publishing

Reply:
Fixed.